lib/sync.js

```diff
@@ -95,6 +97,11 @@ function Sync(src, target, opts, callback) {
     };
   }
 
+  if (opts.live) {
+    this.push.on('complete', self.pull.cancel.bind(self.pull));
+    this.pull.on('complete', self.push.cancel.bind(self.push));
+  }
+
```

Comment:
This is based on top of https://github.com/pouchdb/pouchdb/commit/ada62f35b7fa1badab9dcf8e9b50689dc13e4000 which is already merged